tests/test_all.py

```diff
@@ -46,10 +46,7 @@ def Response_init_new(self, *args, **kwargs):
 
     def _mock_zyte_smartproxy_response(self, url, headers=None, **kwargs):
         zyte_smartproxy_version = choice(("1.36.3-cd5e44", "", None))
-        zyte_smartproxy_headers = {"X-Crawlera-Version": zyte_smartproxy_version}
```

Comment:
Tests are now expected to pass regardless of whether the response contains an X-Crawlera-Version header or any other header.